README.md

```diff
@@ -144,6 +144,11 @@ DISCLAIMER: All below async references were added taking into consideration only
 	* [_Pyright_](https://github.com/microsoft/pyright) is another popular tool for static type checks, more far ahead than _Mypy_ regarding support for newest Python typing features, comes built-in with VS Code Python extension and supports execution in CI technologies, however at time of writing not supported by Pycharm.
 6. <u>Exceptions</u> - [Official docs built-in exceptions reference](https://docs.python.org/3/library/exceptions.html)
 	1. Tip: Derive from _BaseException_ instead of _Exception_ in order to implement an exception type that won't be caught by general purpose _except: Exception_ blocks. This technique is used for cancellations exceptions raised by async/await libraries; general purpose exception handling shouldn't handle cancellations.
+7. ([Conributed by HomiGrotas](https://github.com/HomiGrotas)) <u>Web</u> - Interacting with web services
```

Comment:
1. There's a topic `Http Client ` where `requests` module is more natural, however it's for async I/O only and for reason - blocking I/O is not something I'd recommend for backend/data engineers building data pipelines. We can list the "requests" module as well in same topic and specify explicitly it's not recommended for Production code expected to be scalable, but can be useful for other scenarios. WDYT?
2. BeautifulSoup and Selenium deserve a new topic "web scraping" which doesn't exist yet, great addition! I don't think one should be recommended over the other since they provide different capabilities, I think a quick explanation of how they differ is important for people who don't know them to know which to focus on. BTW notice the typo "Beutifulsoup" should be "BeautifulSoup".
3. Scapy and any other low-level networking aspects I'm not convinced yet are enough general purpose for backend/data engineers. It'd be interesting to open a dedicated issue on that and see if there can be a discussion, WDYT? Meanwhile I propose to add a new topic "Real time networking" or "Low level networking" (I think the latter is more accurate) and add `Scapy` there and other alternatives available.

Reply:
1. Sounds good
2. Ok, will do
3. Ok. It's in issue #4 and changes will be made accordingly